packages/ripple-nuxt-tide/lib/config/tide.config.js

```diff
@@ -79,7 +79,8 @@ const tideConfig = {
           image: {
             field: ['field_block_image', 'field_media_image'],
             filters: ['fieldMediaImage']
-          }
+          },
+          caption: ['field_block_image', 'field_media_caption']
```

Comment:
This can be a filter to take the logic in middleware``` if (!pageData.tidePage.field_show_c_primary_caption) {
        cPrimaryMapping.data.caption = null
      }```

Reply:
Updated.